src/com/intellij/idea/plugin/hybris/codeInspection/rule/beanSystem/BSDuplicateBeanPropertyDefinition.kt

```diff
@@ -57,13 +58,32 @@ class BSDuplicateBeanPropertyDefinition : AbstractBSInspection() {
                 .map { it.properties }
                 .mapNotNull { it[property.name.stringValue] }
 
-            if (otherPropertyDeclarations.size > 1) {
-                holder.createProblem(
-                    property.name,
-                    severity,
-                    displayName
-                )
+                if (otherPropertyDeclarations.size > 1) {
+                    createProblem(holder, property, severity, displayName)
             }
+
+            val beanProperties = mutableMapOf<String, MutableList<Property>>()
+
+            dom.properties.forEach { property ->
+                val currentKey = property.name.toString()
```

Comment:
using `toString` for proxy value is not exactly what should be used here, `.value` or `.stringValue` has to be used instead, btw it may be null